modules/migration-installing-cam-operator-ocp-4.adoc

```diff
@@ -49,7 +50,7 @@ ifdef::sourcecluster-4_1-4_2[]
 . Click *Catalog* -> *OperatorHub*.
 endif::[]
 . On the *OperatorHub* page:
-.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *Cluster Application Migration Operator*.
+.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *CAM Operator*.
```

Comment:
I'm not aware of any name change to 'CAM Operator', it should be 'Cluster Application Migration Operator'.